<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Thomas Vogt</author>
 <class>Arinc665Qt::CompileMediaSetSettingsPage</class>
 <widget class="QWizardPage" name="Arinc665Qt::CompileMediaSetSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="title">
   <string>ARINC 665 XML Compiler Settings</string>
  </property>
  <property name="subTitle">
   <string>Configures the operation.</string>
  </property>
  <attribute name="pageId">
   <string notr="true">0</string>
  </attribute>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Arinc665Qt::ImportMediaSetXmlSettingsWidget" name="mediaSetXmlSettings">
     <property name="title">
      <string>ARINC 665 Media Set XML Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Arinc665Qt::ExportMediaSetSettingsWidget" name="mediaSetOutputSettings">
     <property name="title">
      <string>ARINC 665 Media Set Output Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mediaSetOutputPath">
     <property name="title">
      <string>ARINC 665 Media Set Output Path</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="outputBasePath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectOutputBasePath">
        <property name="icon">
         <iconset theme="document-open">
          <normaloff>../import_media_set_xml</normaloff>../import_media_set_xml</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Arinc665Qt::ImportMediaSetXmlSettingsWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">arinc665_qt/ImportMediaSetXmlSettingsWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Arinc665Qt::ExportMediaSetSettingsWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">arinc665_qt/ExportMediaSetSettingsWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
